### PR TITLE
implemented code coverage feature

### PR DESCRIPTION
**Pull request by pradeexsu**
### implemented code coverage feature for volesti  #139 

[codecov](https://codecov.io/gh/sutharp777/volume_approximation) 
### configured files:
- circleci/config.yml
- CodeCoverage.cmake
- CMakeLists.txt configured
- codecov badge added to README.md file.

### CODE REVIEW

**Comment by pradeexsu** [comment]
@vissarion , @TolisChal I have configured `cicleci/config.yml` but not all, and circleci workflow passes for my repo:branch, what should I do now, please help me out.

**Comment by pradeexsu** [comment]
does it require modifying all the workflow files?

File: examples/CodeCoverage.cmake [hunk 1]
```diff
@@ -0,0 +1,683 @@
+# Copyright (c) 2012 - 2017, Lars Bilke
```
Comment (vissarion) [inline examples/CodeCoverage.cmake:1]: you definitely do not want this file twice (it is also in `test` directory)

**Comment by vissarion** [comment]
> does it require modifying all the workflow files?

It seems that you have an issue in github actions for the `clang`. You may want to consult this https://github.com/marketplace/actions/codecov

**Comment by vissarion** [comment]
You may also want to not include https://github.com/GeomScale/volume_approximation/blob/develop/test/doctest.h in the coverage tests

File: examples/CodeCoverage.cmake (hunk 1 above)
Comment (pradeexsu) [reply examples/CodeCoverage.cmake:1]: ok, I will take care of that

**Comment by pradeexsu** [comment]
ok, I will change it to as expected.

**Comment by vissarion** [comment]
Hi @sutharp777, could you please explain why are you closing this ?

**Pull request closed without merging**